src/client/dispatch.rs

```diff
@@ -497,13 +497,25 @@ fn handle_event<H: EventHandler + Send + Sync + 'static>(
             });
         },
         DispatchEvent::Model(Event::MessageUpdate(mut event)) => {
-            update!(event);
+            let _before = update!(event);
+
+            let _after: Option<Message> = feature_cache! {{
```

Comment:
Although most of the rest of this file uses this technique, we should be able to conditionally compile this binding, see line 264 for an example.